diff --git a/arch/arm/boot/dts/sun8i-v3s-licheepi-zero-dock.dts b/arch/arm/boot/dts/sun8i-v3s-licheepi-zero-dock.dts
index db5cd0b..d20c983 100644
--- a/arch/arm/boot/dts/sun8i-v3s-licheepi-zero-dock.dts
+++ b/arch/arm/boot/dts/sun8i-v3s-licheepi-zero-dock.dts
@@ -62,6 +62,15 @@
 	status = "okay";
 };
 
+&codec {
+	allwinner,audio-routing =
+	"Headphone", "HP",
+	"Headphone", "HPCOM",
+	"MIC1", "Mic",
+	"Mic",  "HBIAS";
+	status = "okay";
+};
+
 &lradc {
 	vref-supply = <&reg_vcc3v0>;
 	status = "okay";
@@ -94,3 +103,10 @@
 		voltage = <800000>;
 	};
 };
+
+&emac {
+	phy-handle = <&int_mii_phy>;
+	phy-mode = "mii";
+	allwinner,leds-active-low;
+  status = "okay";
+};
